scripts/irods/lib.py

```diff
@@ -707,3 +707,26 @@ def add_child_resource(parent_resource_name, child_resource_name, user):
 
 def remove_child_resource(parent_resource_name, child_resource_name, user):
     user.assert_icommand(['iadmin', 'rmchildfromresc', parent_resource_name, child_resource_name])
+
+def get_user_type(session, username):
+    """Get the type field of the user indicated by username."""
+    gql = "select USER_TYPE where USER_NAME = '{}'"
```

Comment:
all four of these in this diff could match more than one user?   `alice#tempZone` and `alice#otherZone`?

Reply:
A fine point. CI tests caught this as well, so I'll tinker with this a bit more.

Reply:
Added the zone names and adjusted tests. Everything is passing now.